playwright.config.ts

```diff
@@ -22,6 +22,12 @@ const TIMEOUT = 5 * 60 * 1000;
 export default defineConfig({
 	timeout: TIMEOUT,
 	workers: DEV ? 5 : 2,
+	expect: {
+		toHaveScreenshot: {
```

Comment:
Can you add some references for those options or some comments about what are their effects?

Reply:
Will do that in my next commit.

Reply:
You mean in this PR right?

Reply:
comments added for the new configs